scripts/upsert-dev-helper.sh

```diff
@@ -0,0 +1,30 @@
+# This script operates like postinstall + preinstall, but for local development
+# builds, where the helper is necessary. Instead of looking for
+# /Applications/Coder Desktop.app, it looks for
+# /Applications/Coder/Coder Desktop.app, which is where the local build is
+# installed.
+
+set -euox pipefail
```

Comment:
```suggestion
set -euxo pipefail
```

It's essentially equivalent to `set -e -u -x -o=pipefail`